frontend/src/app/[locale]/saved-grants/layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { SAVED_GRANTS_CRUMBS } from "src/constants/breadcrumbs";
+import { LayoutProps } from "src/types/generalTypes";
+
+import Breadcrumbs from "src/components/Breadcrumbs";
+import { SessionCheck } from "src/components/user/SessionCheck";
+
+export default function SavedGrantsLayout({ children }: LayoutProps) {
+  return (
+    <>
+      <Breadcrumbs breadcrumbList={SAVED_GRANTS_CRUMBS} />
+      <SessionCheck />
```

Comment:
Any reason not to just do this in the page instead of just creating a new route?

Reply:
my thinking is that since this is about checking the session on the client, doing the check in an isolated component lets us include it in a server component without changing that server component to a client component

Reply:
I meant doing this in the `layout.tsx` instead of the `page.tsx`. Makes sense that it needs to be a client-side and the best way to call that is to use a renderable component.